salt/modules/win_powercfg.py

```diff
@@ -59,172 +64,309 @@ def _get_powercfg_minute_values(scheme, guid, subguid, safe_name):
     else:
         out = split[0]
 
-    raw_settings = re.findall(r"Power Setting Index: ([0-9a-fx]+)", out)
-    return {"ac": int(raw_settings[0], 0) / 60, "dc": int(raw_settings[1], 0) / 60}
+    raw_settings = re.findall(r'Power Setting Index: ([0-9a-fx]+)', out)
+    return {'ac': int(raw_settings[0], 0) / 60,
+            'dc': int(raw_settings[1], 0) / 60}
 
 
 def _set_powercfg_value(scheme, sub_group, setting_guid, power, value):
     '''
-    Sets the value of a setting with a given power (ac/dc) to
-    the given scheme
+    Sets the AC/DC values of a setting with the given power for the given scheme
     '''
+    salt.utils.warn_until(
```

Comment:
Why would we change to something less exact? I'm not clear on the reasoning here. Would it be better just to change the documentation instead?

Reply:
Power settings in the GUI are displayed in Minutes and Hours. As a Windows admin, that's what I would expect to be setting from the CLI as well.

Since the return value is in Minutes and the documentation says minutes, I believe the original intent of the author was to set the time in Minutes.